controls/cis_ocp_1_4_0/section-3.yml

```diff
@@ -26,7 +28,8 @@ controls:
       level: level_1
     - id: 3.2.2
       title: Ensure that the audit policy covers key security concerns
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - audit_profile_set
```

Comment:
Is there a reason not to use this rule for 3.1.1?

Reply:
If there is, please remove the 3.1.1 reference from audit_profile_set

Reply:
Updated to include the rule for control 3.2.1 and keeping the references. I misread the comments in the original profile. Good catch.

Reply:
For 3.2.2, shouldn't we also set `var_openshift_audit_profile` to be `WriteRequestBodies`?

Reply:
Could we set the `var_openshift_audit_profile` to `WriteRequestBodies` in the CIS profile?

Reply:
Good catch, setting that as a profile-level variable in https://github.com/ComplianceAsCode/content/pull/10833